webpack.config.babel.js

```diff
@@ -4,6 +4,7 @@ import MiniCssExtractPlugin from "mini-css-extract-plugin";
 import Uglify from "terser-webpack-plugin";
 import CopyPlugin from "copy-webpack-plugin";
 
+import webpack from 'webpack';
```

Comment:
@n1313 nitpick: move this import into the first block please. second block is for relative-imports

our eslint-runner doesn't check this file automatically :-/

Reply:
Sorry, my bad. Done